packages/http-smoke-testing/tests/Unit/RouterAdapter/SymfonyRouterAdapterTest.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace Tests\HttpSmokeTesting\Unit\RouterAdapter;
+
+use Doctrine\Common\Annotations\AnnotationReader;
+use PHPUnit\Framework\TestCase;
+use Shopsys\HttpSmokeTesting\Annotation\DataSet;
+use Shopsys\HttpSmokeTesting\Annotation\Skipped;
+use Shopsys\HttpSmokeTesting\RouteInfo;
+use Shopsys\HttpSmokeTesting\RouterAdapter\SymfonyRouterAdapter;
+use Shopsys\HttpSmokeTesting\Test\TestController;
+use Symfony\Bundle\FrameworkBundle\Routing\AnnotatedRouteControllerLoader;
+use Symfony\Component\Routing\Router;
+
+class SymfonyRouterAdapterTest extends TestCase
+{
+    public function testGetAllRouteInfoExtractsInformationFromRouteCollection()
+    {
+        $router = new Router(
+            new AnnotatedRouteControllerLoader(new AnnotationReader()),
+            TestController::class
+        );
+
+        $adapter = new SymfonyRouterAdapter($router);
+
+        $routeInfos = $adapter->getAllRouteInfo();
+
+        self::assertCount(2, $routeInfos);
+
+        self::assertInstanceOf(RouteInfo::class, $routeInfos[0]);
+        self::assertSame('/hello/{name}', $routeInfos[0]->getRoutePath());
+        self::assertCount(2, $routeInfos[0]->getAnnotations());
+        self::assertInstanceOf(DataSet::class, $routeInfos[0]->getAnnotations()[0]);
```

Comment:
I'm missing asserts for parameters' values.

Reply:
`done();`